server/.env

```diff
@@ -1,2 +1,3 @@
 TEAM_NAME="Shums"
 FLASK_ENV="development"
+DATABASE_URL="postgresql://localhost/team_broccoli"
```

Comment:
Let's gitignore this file in this PR too, just in case in the future we add something more private

Reply:
Removed .env and added .env.example. Added .env to .gitignore.